src/utils.js

```diff
@@ -0,0 +1 @@
+export const sanitizeNumber = (number) => number.toString().trim().replace(' ', '');
```

Comment:
Payment did this sanitization internally so I needed to add it here to pass the sanitized value to the `creditCardType` function

Reply:
Merge it with the `card` module

Reply:
done